css/style.css

```diff
@@ -16,4 +15,110 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+body {
+  background-color: white;
+  margin: 0;
+  text-align: center;
+}
+
+header {
+  display: flex;
+  padding: 20px 0px;
+  justify-content: space-between;
+}
+
+header img {
+  height: 50px;
+  padding-left: 80px;
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 100px;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  font-size: 18px;
+}
+
+li {
+  padding: 0 25px;
+}
+
+a {
+  color: #7f8c8d;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #e74c3c;
+}
+
+.meet-karma {
+  font-weight: bold;
+}
+
+article {
+  padding: 100px;
+}
+
+.hero {
+  background: url(../img/first-background.jpg) no-repeat;
+  background-size: cover;
+  height: 250px;
```

Comment:
You might find that using internal padding on the hero is a more flexible way of controlling its size, than setting a fixed height. It won't make too much of a difference now, but when we start adding responsiveness to the page, it might make things easier :)